cmd/pipeline/destroy.go

```diff
@@ -72,16 +73,17 @@ func destroy(ctx context.Context) *cobra.Command {
 	cmd.Flags().StringVarP(&name, "name", "p", "", "name of the pipeline (defaults to the folder name)")
 	cmd.Flags().StringVarP(&namespace, "namespace", "n", "", "namespace where the up command is executed (defaults to the current namespace)")
 	cmd.Flags().BoolVarP(&wait, "wait", "w", false, "wait until the pipeline finishes (defaults to false)")
+	cmd.Flags().BoolVarP(&destroyVolumes, "volumes", "v", false, "remove persistent volumes created by the pipeline (defaults to false)")
```

Comment:
`-d` or `-r` instead of `-v` since the action is destroy volumes or remove volumes? (not sure if we have something similar in other commands)

Reply:
yes, we are using `-v` in other places.
also, people are already familiar with `-v` usage in docker/docker-compose CLIs